.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+# yamllint --format github .github/workflows/test.yml
+---
+name: Deploy master and release versions
+
+# We deploy on master and release versions, regardless of if the commit is
+# documentation-only or not.
+on:
+  push:
+    tags: '[0-9]+.[0-9]+.[0-9]+'
+    branches: master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          # license-maven-plugin needs the full history to generate copyright
+          # year range. Ex. 2013-2020. Don't do a shallow clone!
+          fetch-depth: 0
+      - name: Cache docker
+        uses: actions/cache@v2
+        with:
+          path: ~/.docker
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/Dockerfile') }}
+          restore-keys: ${{ runner.os }}-docker
+      - name: Configure Deploy
+        run: build-bin/configure_deploy
+        with:
+          # GH_USER=user_that_created_GH_TOKEN
+          GH_USER: ${{ secrets.GH_USER }}
+          # GH_TOKEN=XXX-https://github.com/settings/tokens-XXX
```

Comment:
I actually can't parse this comment at all - doesn't GitHub UI show the token after creating it and you copy-paste? This XXX doesn't refer to anything and is misleading I think.

Reply:
was trying to avoid conflation with the URL literal. any other ideas?

Reply:
spiked a try! agree it is weird..